src/App.js

```diff
@@ -15,10 +17,22 @@ export default function App() {
             renders the first one that matches the current URL. */}
         <Switch>
           <Route path="/translations">
-            <SearchPage />
+            <div>
+              <HeaderBar />
```

Comment:
Not entirely sure how, but could you consider a way to move the header bar component to be outside the switch somehow? so regardless of the route the header would be there?

Reply:
Good point! I've changed the code so that the header bar component is outside the switch now.